### DISCUSSION TITLE
30.01.2024 - Kivy

### DISCUSSION THREAD

**Discussion by DGrillCoder** [grilldb97/legendary-space-zebra#23] [Show and tell]
Ich habe mir diese Woche genauer mit Kivy beschäftigt. Das ist ein Packet, womit man Apps erstellen kann
23.01.2024: kivy: BoxLayout
24.01.2024: kivy: AnchorLayout
25.01.2024: kivy: GridLayout
26.01.2024: kivy: StackLayout
27.01.2024: kivy: ScrollView mit StackLayout
28.01.2024: kivy: PageLayout - mit dem Finger nach links wischen und dann kommen die einzelnen Seiten
29.01.2024: kivy: ButtonClick
30.01.2024: kivy: ClickCounter

Nacheinander habe ich die Layouts angeschaut und umgesetzt. Diese können auch kombiniert werden.
An folgendem YouTube Video habe ich mich orientiert: https://youtu.be/l8Imtec4ReQ

Es ist faszinierend, wie viel und schnell man mit Kivy machen kann. In wenigen Minuten hat man seine App.
Ich bin gespannt, was man da noch alles machen kann. Ich kanns nur empfehlen. Schaut es euch an und lasst euch inspirieren

**Comment by katharinaheller** [comment]
Klingt spannend! Ich habe bisher wenig von Kivy gehört, aber deine Erfahrungen machen mich neugierig. Es ist eine coole Möglichkeit, schnell und einfach plattformübergreifende Apps zu entwickeln. Das erkennt man vor allem an deinen Programmierprojekten. Weiter so! :)

**Comment by SirPatschiii** [comment]
Kiwi, can you eat it? 😂
No just joking, exciting topic. It's cool that there are packages that make it so easy to develop an app.
